pyproject.toml

```diff
@@ -0,0 +1,13 @@
+[build-system]
+# Minimum requirements for the build system to execute.
+requires = [
+    "setuptools",
+    "wheel",
+    "Cython>=0.29.13",  # Note: sync with setup.py
+    "numpy==1.13.3; python_version=='3.5' and platform_system!='AIX'",
```

Comment:
These were copied from scipy. I'm not sure what the platform_system stuff is for.

Reply:
Not sure if it matters but we technically required `3.5.3` as a minimum

Reply:
Do we have any idea if pandas builds at all on AIX ?

> Not sure if it matters but we technically required 3.5.3 as a minimum

This line does not control the python version, it is only saying that if you have python 3.5, you need that numpy. 

But that said, I am wondering if there is a way to specify python_requires as well in the pyproject.toml world.

Reply:
I suspect not, since this is invoked by pip, and pip is tied to a particular python?

Reply:
@ayappanec do you happen to know if pandas can be built / used with AIX? We're trying to determine if these workarounds you added to SciPy's pyproject.toml are needed in ours as well.

Reply:
But that pip with a specific python still needs to know if it can build/install pandas or not (depending on the python version). Currently this information is in setup.py, but I thought pyproject.toml would allow (if not using setuptools, but eg flit) having no setup.py. 

Anyway, that's not an important issue for this PR ;)

Reply:
Numpy 1.14.0 ( and from that level onwards ) has a patch that is required for building projects like Scipy (which depends on numpy's script for gfortran flags) in AIX . So if Pandas also has the same approach , then we need to have these same conditions.

Reply:
Thanks. Pandas has NumPy as a build dependency, but we don't do anything with fortran. So it sounds like we're safe to remove?

Reply:
Okay. But still i feel like having these conditions doesn't do any harm i guess. So better we will have those on the safer side.

Reply:
> Since we are targeting 1.0 now I guess can remove the line for 3.5 support

Master still has 3.5 support I think?

Reply:
Yeah, I don't know that we've discussed dropping python 3.5 for 1.0. I think the idea was to follow the NEP policy if that gets done in time (I think it's supposed to be merged this week?)

Reply:
yeah, thought we should prob remove for 3.5

Reply:
> Yeah, I don't know that we've discussed dropping python 3.5 for 1.0. I think the idea was to follow the NEP policy if that gets done in time (I think it's supposed to be merged this week?)

https://github.com/pandas-dev/pandas/issues/27492#issuecomment-513545165